### PR TITLE
Fix/wp deletion

### PR DESCRIPTION
**Pull request by ulferts**
OP ticket: https://www.openproject.org/issues/2400

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -271,14 +271,10 @@
...
+    resource :bulk, :controller => 'bulk', :only => [:edit, :update, :destroy]
   end
 
-  namespace :work_package_bulk do
-    get :edit, :format => false
-    put :update, :format => false
-  end
-
-  resources :work_packages, :only => [:show, :edit, :update, :index] do
+  resources :work_packages, :only => [:show, :edit, :update, :index, :destroy] do
```
Comment (sschu) [inline config/routes.rb]: I think destroy should be removed here.

File: spec/permissions/delete_work_packages_spec.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+require 'spec_helper'
+require_relative  '../support/permission_specs'
+
+describe WorkPackages::BulkController, "delete_work_packages permission", type: :controller do
+  include PermissionSpecs
+
+  check_permission_required_for('work_packages/bulk#destroy', :delete_work_packages)
+end
+
+describe WorkPackagesController, "delete_work_packages permission", type: :controller do
```
Comment (sschu) [inline spec/permissions/delete_work_packages_spec.rb]: This can also be removed probably.

File: spec/routing/work_packages_spec.rb [hunk 3]
```diff
@@ -104,4 +104,10 @@
                                              :action => 'update',
                                              :id => '1' )
   end
+
+  it "should connect DELETE /work_packages/1 to work_packages#destroy" do
```
Comment (sschu) [inline spec/routing/work_packages_spec.rb]: Finally, this is also a candidate for deletion.

**Comment by ulferts** [comment]
@sschu, rebased the branch in order to reestablish mergeability. Fixed the routes along the way.

**Comment by sschu** [comment]
:+1:

**Pull request merged**